server.js

```diff
@@ -1,5 +1,6 @@
 //node packages
 const SMTPServer = require("simple-smtp-listener").Server;
+const axios = require("axios");
```

Comment:
Why `axios` and not the built-in [`https`](https://nodejs.org/api/https.html)?

Reply:
Chorebot uses axios so I tried to keep it consistent. I'm happy to switch to the built in https client.